src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -282,6 +283,12 @@ static MvcCoreLoggerExtensions()
                 LogLevel.Debug,
                 2,
                 "Applied the configured form options on the current request.");
+
+            _autoValidateModelFilterExecuting = LoggerMessage.Define(
+                LogLevel.Debug,
+                1,
+                "AutoValidateModeFilter returned a BadRequestObjectResult since the ModelState is invalid.");
```

Comment:
This should be like `The request has model state errors, returning an error response`.